packages/std/src/imports.rs

```diff
@@ -16,12 +18,17 @@ static ADDR_BUFFER: usize = 72;
 // This interface will compile into required Wasm imports.
 // A complete documentation those functions is available in the VM that provides them:
 // https://github.com/confio/cosmwasm/blob/0.7/lib/vm/src/instance.rs#L43
-//
-// TODO: use feature switches to enable precompile dependencies in the future,
-// so contracts that need less
 extern "C" {
     fn read_db(key: *const c_void, value: *mut c_void) -> i32;
     fn write_db(key: *const c_void, value: *mut c_void);
+    fn remove_db(key: *const c_void);
+
+    // scan creates an iterator, which can be read by consecutive next() calls
+    #[cfg(feature = "iterator")]
+    fn scan(start: *const c_void, end: *const c_void, order: i32) -> i32;
+    #[cfg(feature = "iterator")]
+    fn next(key: *mut c_void, value: *mut c_void) -> i32;
```

Comment:
Looking at the list now, it feels like a `db_` prefix (as in namespacing) would be a good idea.

```
db_read
db_write
db_remove
db_scan
db_next
```

Reply:
Shall I do that as part of this PR?
Or maybe a follow up PR (as it renames existing functions as well)?

Reply:
For now, I just added a `_db` suffix to `scan` and `next` for consistency with current pattern. Happy if you rename them all later.

Reply:
👍 Whatever we can separate out is good. Otherwise it becomes a merging nightmare at some point. But such a big addition raises awareness to many things that can be improved.

Reply:
https://github.com/CosmWasm/cosmwasm/issues/191